File: vue-coder/src/main/java/org/foolcoder/common/Logger.java
package org.foolcoder.common;

import java.text.DateFormat;
import java.util.Date;

public class Logger {

    public static void log(String operation) {
        log(operation, "");
    }

    public static void log(String operation, String content) {
        DateFormat df = DateFormat.getDateTimeInstance();
        String time = df.format(new Date());
        System.out.println(time + Constants.LOG_SPACE + operation + Constants.LOG_LINE + content);
    }

}

File: src/common/Logger.js
const { LOG_LINE, LOG_SPACE } = require('./Constants')

const log = (operator, content = '') => {
  const time = new Date().toLocaleString()
  if (content) {
    console.log(`${time}${LOG_SPACE}${operator}${LOG_LINE}\n`, content)
  } else {
    console.log(`${time}${LOG_SPACE}${operator}:`)
  }
}

const error = (operator, content = '') => {
  const time = new Date().toLocaleString()
  if (content) {
    console.error(`${time}${LOG_SPACE}${operator}${LOG_LINE}\n`, content)
  } else {
    console.error(`${time}${LOG_SPACE}${operator}:`)
  }
}

module.exports = { log, error }
